src/EnergyPlus/DataGlobals.hh

```diff
@@ -164,6 +164,7 @@ struct DataGlobal : BaseGlobalStruct
     bool CountNonZoneEquip = true;
     int FDsimDay = 0;
     int FDnumIterYears = 0;
+    bool printConsoleOutput = true;
```

Comment:
A new state variable for specifying whether we should print messages to the console.  For normal EnergyPlus operation this will be true, but now this can be toggled via API.  This makes it nice to mute the console output while running as a library.